integr8ly/tutorial-web-app
# Resolves missing Solution Explorer logo

## What
 Resolves missing Solution Explorer logo in Firefox.

## Why
See [INTLY-6771](https://issues.redhat.com/browse/INTLY-6771 ).


## How
 Added `width` properties to the SVGs, and re-exported a corrupt SVG. 

## Verification Steps

See screenshots below, or pull down branch locally.

<img width="1680" alt="Screen Shot 2020-05-06 at 4 30 09 PM" src="https://user-images.githubusercontent.com/32821331/81225351-186ff900-8fb7-11ea-8226-288dace90a42.png">

<img width="1680" alt="Screen Shot 2020-05-06 at 4 30 47 PM" src="https://user-images.githubusercontent.com/32821331/81225362-1b6ae980-8fb7-11ea-96ed-909c02e58b74.png">


## Checklist:


## Progress


## Additional Notes
